community/opendkim/opendkim.initd

```diff
@@ -1,16 +1,17 @@
 #!/sbin/openrc-run
 
+owner=opendkim
 pidfile=/run/opendkim/opendkim.pid
-
+cfgfile=/etc/opendkim/opendkim.conf
 command=/usr/sbin/opendkim
-command_args="-f ${EXTRA_OPTS}"
-command_background=yes
+command_args="$command_args -u $owner -P $pidfile"
```

Comment:
Oh, I thought exactly the opposite. Everywhere I tried to make the service itself generate pidfile.
bad news for me ... ((